nf_core/pipeline-template/{{cookiecutter.name_noslash}}/bin/scrape_software_versions.py

```diff
@@ -18,11 +19,14 @@
 
 # Search each file using its regex
 for k, v in regexes.items():
-    with open(v[0]) as x:
-        versions = x.read()
-        match = re.search(v[1], versions)
-        if match:
-            results[k] = "v{}".format(match.group(1))
+    if os.path.exists(v[0]):
```

Comment:
Nice!

If I'm being pedantic, the more pythonistic way of doing this would be just to catch the error if we trigger an exception - no additional import needed and fits with the [EAFP](https://docs.python.org/3.4/glossary.html#term-eafp) python style: _Easier to ask for forgiveness than permission_.

But this is so minor and the import is in the standard lib, so really - who cares..! 😅

Reply:
No probs. Fixed 👍